plugins/woocommerce/client/admin/client/settings-payments/settings-payments-main.scss

```diff
@@ -57,6 +57,13 @@
 		}
 	}
 
+	// Settings specific style override for the pill component.
+	.woocommerce-pill {
```

Comment:
p.s. I have also version of this in https://github.com/woocommerce/woocommerce/pull/52161/files#diff-02a7ac5265a00491ebc0bd6aa4218770f380088d62c3263c5a18e2396b8bdb8eR77
😅

Reply:
```
background: var(--Alias-bg-bg-surface-secondary, #f5f5f5);
border: none;
border-radius: var(--radius-primitive, 4px);
```
it's copied from Figma

Reply:
Oh, great! That's fine, I guess your PR will get merged first so I'll resolve this if a conflict occurs.

Reply:
merged my PR!

Reply:
Thanks. I removed my styling here!